W-10205079 Add open source files + min cci config

spelak-salesforce opened the pull request:
[W-10205079](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000QlELlYAN/view)

Adding open source files:
- `.github/CODEOWNERS`
- `CODE_OF_CONDUCT.md`
- `LICENSE`
- `SECURITY.md`

Adding minimal `cumulusci.yml` configuration.

# Critical Changes

# Changes
Adding open source files and minimal CumulusCI configuration.

# Issues Closed

# Critical Changes

# New Metadata

# Deleted Metadata

# Definition of Done
Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:
- [ ] Any net new LWC work has JEST test coverage 50% or above
- [ ] Default Sa11y tests pass for all LWC components
- [ ] 🔒 Secure both Front-end (LWC) & back-end (Apex) as necessary
- [ ] 🔑 Grant users access in Permission Sets (Object, Field, Apex Class) as necessary
- [X] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
- [X] Make sure that ACs are updated (if any gaps)
- [X] **All acceptance criteria have been met**
    - [X] Developer
    - [X] Code Reviewer
- [X] Pull Request contains draft release notes
- [ ] Labels, help text, and customer facing messages are reviewed by Docs
- [X] QE story level testing completed

spelak-salesforce committed:
W-10205079 Add open source files + min cci config

Adding open source files:
- .github/CODEOWNERS
- CODE_OF_CONDUCT.md
- LICENSE
- SECURITY.md

Adding minimual `cumulusci.yml` configuration.

spelak-salesforce committed:
Add open source license

Adding open source license to `cumulusci.yml`.

spelak-salesforce committed:
Always use sfdobase version 1.0

See [W-10046810](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000HOtb5YAD/view)

spelak-salesforce committed:
Adding license header + gitignore

Adding the `.gitignore` used in Grants-Management.
Adding `LICENSE.header.txt` as a reference to inculded in supported source code files.

a-contreras commented on LICENSE.header.txt:
```diff
@@ -0,0 +1,4 @@
+Copyright (c) 2021, salesforce.com, inc.
```
Why `LICENSE.header.txt` and not just `LICENSE.txt ` as recommended here: https://github.com/salesforce/oss-template/blob/master/license_info.md ?

spelak-salesforce replied:
I see other open source repos using just `LICENSE` + VS Code uses a special 🔑  icon next to license.  Cool 😎 , but not very good reasons.

![license-icon](https://user-images.githubusercontent.com/46537641/143064611-ea3ecca3-6e02-480e-a5f5-34a377de3d9c.png)

It seems it's [ok to use `LICENSE`, `LICENSE.txt`, or `LICENSE.md`](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository), but it's most common to have `LICENSE.txt`.  I renamed the license file to `LICENSE.txt` to comply.

![licnse txt-icon](https://user-images.githubusercontent.com/46537641/143064763-ef5283db-414e-4942-b8b3-03ee977516fd.png)

spelak-salesforce committed:
Renaming to LICENSE.txt

To comply with the OSS request instructions, renaming `LICENSE` file to `LICENSE.txt`.

a-contreras reviewed (approved):
All the changes look good!
